bliss/generate.py

```diff
@@ -24,7 +24,7 @@
 def generate(cfg: DictConfig):
     max_images_per_file = cfg.generate.max_images_per_file
     cached_data_path = cfg.generate.cached_data_path
-    files_start_idx = cfg.generate.files_start_idx
+    files_start_idx = cfg.generate.process_index * cfg.generate.num_files_per_process
```

Comment:
Following the above, we could then change this to

```python
# code from few lines below, can be shifted up; or these two new lines shifted down
n_files = -(cfg.generate.n_batches * bs // -images_per_file) # note: this is technically "n_files for this process"

process_index = cfg.generate.process_index or 0
files_start_idx = process_index * n_files
```

Reply:
Seems good but probably better done as a separate PR

Reply:
Oh I see, you're proposing hiding `process_index` from the user?

Reply:
That's easy enough. I got confused by the `n_files = ...` line but I see that's already in the code.

Reply:
Yup, thanks!